h/templates/deform/sequence_item.jinja2

```diff
@@ -1,15 +1,27 @@
 {% if not field.widget.hidden %}
 <li class="list-input__item" title="{{ field.description }}">
 
-  {{ field.serialize(cstruct=cstruct) | safe }}
+  <div class="list-input__item-inner">
+    {{ field.serialize(cstruct=cstruct) | safe }}
 
-  <button class="btn btn--danger list-input__remove-btn" type="button"
-          data-ref="removeItemButton">Remove</button>
+    {% if (can_delete is not defined) or can_delete %}
```

Comment:
`can_delete` is undefined when rendering the template for new field items, because Deform doesn't allow passing additional arguments to be used when rendering the `sequence_item` template in this case.